src/ApplyOfflineMap.cpp

```diff
@@ -282,6 +311,42 @@ try {
 		AnnounceStartBlock("Applying first offline map to data");
 	}
 */
+	
+		DataArray3D<int> dataGLLNodesIn;
+		DataArray3D<double> dataGLLJacobianIn;
+		
+		DataArray3D<int> dataGLLNodesOut;
+		DataArray3D<double> dataGLLJacobianOut;
+		Mesh OverlapMesh;
+		Mesh SourceMesh;
+			
+		//FV or FE input mesh
+		if(strInputMesh != ""){
+			Mesh meshSource(strInputMesh);
+			SourceMesh = meshSource;
```

Comment:
This is a copy-by-value statement. No real reason to do this in the current context. Please unify the usage of `meshSource` to `SourceMesh` with appropriate scope.